tests/c/bfbug.c

```diff
@@ -0,0 +1,165 @@
+// Run-time:
+//   env-var: YKD_SERIALISE_COMPILATION=0
+//   env-var: YKD_PRINT_JITSTATE=1
+//   stderr:
+//     ...
+//     jit-state: enter-jit-code
+//     jit-state: exit-jit-code
+//     ...
+//     jit-state: enter-stopgap
+//     ...
+//   stdin:
+//     1234
+
+// This is bf_base.c from https://github.com/ykjit/ykcbf modified to hard-code the input to the
```

Comment:
Just realised: this is probably based on `yk_simple_bf.c` not `bf_base.c`?

Reply:
Fixed in [c4ec947](https://github.com/ykjit/yk/pull/552/commits/c4ec947b6c1b9d4c9cb7ee9f795f92c1492a2a71)

Reply:
Works for me, though the commented line is now very long?

Reply:
Shortened in https://github.com/ykjit/yk/pull/552/commits/01ddf7364f7133b9134b2b6d978a6a076d815a73

And while I was there I reformatted too which I hadn't done yet.